README.md

```diff
@@ -1 +1,173 @@
+
+## Property Attributes
+
+In addition to the main dataset we also released a property attributes dataset, `properties.tsv.gz`. 
+
+Property attributes would likely boost predictive performance of proposed approaches. These attributes would allow us
+to go beyond a list of identifiers and generalize over *similar* properties.
+
+| Attribute Name | DataType       | Description                                                                                  |
+|----------------|----------------|----------------------------------------------------------------------------------------------|
+| prop_id        | Long           | It matches prop_id from sequences.                                                           |
+| type           | Integer        | The property type (anonymized); less frequent property types are merged into one indicator. |
+| star_rating    | Integer        | The star rating of the hotel, from 1 to 5.  A null indicates the property has no stars, the star rating is not known or cannot be publicized. |
+| review_rating  | Integer        | The mean customer review score for the property on a scale out of 5, rounded to nearest integers. A null means there have been no reviews. |
+| location_score | Float          | Location scores, the higher the score the better location of a property. |
+| amenity_airconditioning | Boolean ||
+| amenity_airporttransfer | Boolean ||
+| amenity_bar | Boolean ||
+| amenity_freeairporttransportation | Boolean||
+| amenity_freebreakfast | Boolean||
+| amenity_freeparking | Boolean||
+| amenity_freewifi | Boolean||
+| amenity_gym | Boolean||
+| amenity_highspeedinternet | Boolean||
+| amenity_hottub | Boolean||
+| amenity_laundryfacility | Boolean ||
+| amenity_parking | Boolean ||
+| amenity_petsallowed | Boolean ||
+| amenity_privatepool | Boolean ||
+| amenity_spaservices | Boolean ||
+| amenity_swimmingpool | Boolean ||
+| amenity_washerdryer | Boolean ||
+| amenity_wifi | Boolean ||
+
+# Evaluation
+
+The submitted models will be evaluated on *hits@k* metric which is defined as the number of times the next clicked 
+item appears at the top k predicted items. We will set k=5.
+
+# Baselines
+
+We release an implementation of a simple baseline approach as well as the evaluation *hits@k* metric. The baseline 
+implements a simple *Markov model* that calculates the transition matrix based on a window of size 1 to 
+calculate the transition probabilities. During prediction the baseline uses only the last click in the session to 
+propose the next clicked hotel. The baseline and evaluation metric are implemented in Python and Tensorflow.
+
+In order to run the baseline you can use the following command:
+```sh
+python baseline_evaluator.py --train_path path_to_train_file --eval_path path_to_test_file
+```
+
+## Software requirements
+
+The code has been tested with `Python 3.8` and requires `Tensorflow>=2.0`, `pandas>=1.4`, `numpy>=1.18` and 
```

Comment:
Here is TF 2.7.1 we have in the requirements

Reply:
2.7.1>=2.0 so it should be ok ?